rustup-1.83.0.json

```diff
@@ -0,0 +1,101 @@
+{
+    "name": "rustup",
```

Comment:
why do you need rustup? There is a whole rust toolchain available

Reply:
[CargoKit](https://github.com/irondash/cargokit) which is used by Flutter libraries using Rust requires `rustup` as stated in the [architecture doc](https://github.com/irondash/cargokit/blob/main/docs/architecture.md).

Reply:
I found a few related issues for cargokit:

- https://github.com/irondash/cargokit/issues/86
- https://github.com/irondash/cargokit/issues/100
- https://github.com/irondash/cargokit/issues/105

Would you recommend I explore the `rust-toolchain.toml` route or is it okay to use rustup here with this explanation?